openassessment/xblock/openassessmentblock.py

```diff
@@ -246,6 +247,23 @@ class OpenAssessmentBlock(MessageMixin,
         help="Indicates whether or not there are peers to grade."
     )
 
+    label_list = List(
+        default = DEFAULT_LABEL_LIST,
+        scope = Scope.content,
+        help="List of strings relavent to the content."
+    )
+
+    @property
+    def labels(self):
+        return ",".join(self.label_list)
+
+    @labels.setter
+    def labels(self, value):
+        label_list = [label.strip() for label in value.split(",")]
+
+        # Remove duplicates and store.
```

Comment:
Convert this inline comment into docstring.

Reply:
Done.